src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -23,6 +23,99 @@ public static partial class EdgeCasesTests
     {
         [Fact]
         [OuterLoop(/* Leaks key on disk if interrupted */)]
+        public static void ImportEdgeCase()
```

Comment:
I’m not sure a “better” name would fit to publish in public ☺, but I’ll think about it.